src/main/java/org/terasology/thirst/ThirstAuthoritySystem.java

```diff
@@ -67,6 +91,13 @@ public void drinkConsumed(ActivateEvent event, EntityRef item, DrinkComponent dr
         }
     }
 
+    /**
+     * Updates the thirst attribute of the character upon movement.
```

Comment:
It may be useful to also describe that the intention is to get thirsty while moving.